examples/network_functions/go-gtp/pgw/pgw_default.yml

```diff
@@ -1,6 +1,5 @@
 local_addresses:
   s5c_ip: "172.25.1.14"
   s5u_ip: "172.25.0.14"
-  sgi_ip: "10.0.1.254"
-sgi_if_name: "eth3"
+sgi_if_name: "eth2"
```

Comment:
Same as with enb_default.yml

Reply:
Let me change the order of the `docker-compose.yml` file to avoid these changes